eth/handler.go

```diff
@@ -437,14 +437,14 @@ func (h *handler) BroadcastTransactions(txs types.Transactions, propagate bool)
 	// Broadcast transactions to a batch of peers not knowing about it
 	if propagate {
 		for _, tx := range txs {
-			peers := h.peers.ethPeersWithoutTransacion(tx.Hash())
+			peers := h.peers.ethPeersWithoutTransaction(tx.Hash())
 
 			// Send the block to a subset of our peers
 			transfer := peers[:int(math.Sqrt(float64(len(peers))))]
 			for _, peer := range transfer {
 				txset[peer] = append(txset[peer], tx.Hash())
 			}
-			log.Trace("Broadcast transaction", "hash", tx.Hash(), "recipients", len(peers))
+			log.Trace("Broadcast transaction", "hash", tx.Hash(), "recipients", len(txset))
```

Comment:
That's not a typo fix -- pretty certain that changes the semantics of the message

Reply:
Yes, it changes the message from logging all peers to only logging the peers that we actually send the transaction to

Reply:
Not quite, does it? The `len(txset)` will grow for each transaction.. ? 
So if `txs` are two transactoins, and tx1 is sent to `peerA`, then txset will have one key, `peerA`. 
If `tx2` is sent only to `peerB`, then the key `peerB` will be added to `txset`, and `len(txset)` will be `2`, not `1`. 
I think what you want is `len(transfer)`

Reply:
Ahh I meant to put `len(transfer)`, sorry